### ISSUE TITLE
Store the locale code in firmware EEPROM instead

### ISSUE THREAD

**Issue by valeriansaliou** [makers-for-life/makair-control-ui#78]
The locale code should be stored in the firmware EEPROM itself, that way the Control UI can be ran on a read-only partition, which is much better in terms of durability.

A telemetry protocol update should be done for this to be possible.

**Comment by dsferruzza** [comment]
Depending on how we can manipulate the EEPROM, it may or may not be useful to:

- use a versioned "schema" of the data we store
- use a migration system for when we need to change the schema while keeping the data
- decide what API we expose through the telemetry/control protocol
- provide a way to do a hard reset of the EEPROM (via one of the maintenance buttons?)

**Comment by valeriansaliou** [comment]
Right, those are all good ideas. All those would be abstracted from the UI by the firmware & telemetry protocol.

**Comment by valeriansaliou** [comment]
Ref https://github.com/makers-for-life/makair-telemetry/commit/af57f23463d0b5385b89c0fe7152f420c1e3f599
